pdr_backend/sim/dash_plots/callbacks.py

```diff
@@ -69,12 +71,22 @@ def update_graph_live(n, selected_vars, selected_vars_old):
         except Exception as e:
             return [get_waiting_template(e)]
 
-        elements = get_header_elements(run_id, st, ts)
+        header = get_header_elements(run_id, st, ts)
+        elements = []
 
         state_options = sim_plotter.aimodel_plotdata.colnames
         elements.append(selected_var_checklist(state_options, selected_vars_old))
 
         figures = get_figures_by_state(sim_plotter, selected_vars)
-        elements = elements + arrange_figures(figures)
-
-        return elements
+        aranged_figures = arrange_figures(figures)
```

Comment:
let's do a different arrangement. There's no point in keeping the old one if it doesn't apply. And actually I would transform it into a dict (key is category/tab, value is a list of figures)

Reply:
then tabs would also be much easier to form